preprocessing/nextclade/tests/test_data/cchf_1/metadata.tsv

```diff
@@ -0,0 +1,2 @@
+accessionVersion	accession	version	submissionId	isRevocation	submitter	groupName	groupId	submittedAtTimestamp	submittedDate	releasedAtTimestamp	releasedDate	dataUseTerms	dataUseTermsRestrictedUntil	dataUseTermsUrl	versionStatus	versionComment	pipelineVersion	ampliconPcrPrimerScheme	ampliconSize	anatomicalMaterial	anatomicalPart	authorAffiliations	authors	bioprojectAccession	biosampleAccession	bodyProduct	breadthOfCoverage	cellLine	collectionDevice	collectionMethod	completeness_L	completeness_M	completeness_S	consensusSequenceSoftwareName	consensusSequenceSoftwareVersion	cultureId	dehostingMethod	depthOfCoverage	diagnosticMeasurementMethod	diagnosticMeasurementUnit	diagnosticMeasurementValue	diagnosticTargetGeneName	diagnosticTargetPresence	displayName	earliestReleaseDate	environmentalMaterial	environmentalSite	experimentalSpecimenRoleType	exposureDetails	exposureEvent	exposureSetting	foodProduct	foodProductProperties	frameShifts_L	frameShifts_M	frameShifts_S	gcaAccession	geoLocAdmin1	geoLocAdmin2	geoLocCity	geoLocCountry	geoLocLatitude	geoLocLongitude	geoLocSite	hostAge	hostAgeBin	hostDisease	hostGender	hostHealthOutcome	hostHealthState	hostNameCommon	hostNameScientific	hostOriginCountry	hostRole	hostTaxonId	hostVaccinationStatus	insdcAccessionBase_L	insdcAccessionBase_M	insdcAccessionBase_S	insdcAccessionFull_L	insdcAccessionFull_M	insdcAccessionFull_S	insdcRawReadsAccession	insdcVersion_L	insdcVersion_M	insdcVersion_S	isLabHost	length_L	length_M	length_S	ncbiReleaseDate	ncbiSourceDb	ncbiSubmitterCountry	ncbiUpdateDate_L	ncbiUpdateDate_M	ncbiUpdateDate_S	ncbiVirusName	ncbiVirusTaxId	passageMethod	passageNumber	presamplingActivity	previousInfectionDisease	previousInfectionOrganism	purposeOfSampling	purposeOfSequencing	qualityControlDetails	qualityControlDetermination	qualityControlIssues	qualityControlMethodName	qualityControlMethodVersion	rawSequenceDataProcessingMethod	referenceGenomeAccession	sampleCollectionDate	sampleCollectionDateRangeLower	sampleCollectionDateRangeUpper	sampleReceivedDate	sampleType	sequencedByContactEmail	sequencedByContactName	sequencedByOrganization	sequencingAssayType	sequencingDate	sequencingInstrument	sequencingProtocol	signsAndSymptoms	specimenCollectorSampleId	specimenProcessing	specimenProcessingDetails	stopCodons	totalAmbiguousNucs_L	totalAmbiguousNucs_M	totalAmbiguousNucs_S	totalDeletedNucs_L	totalDeletedNucs_M	totalDeletedNucs_S	totalFrameShifts_L	totalFrameShifts_M	totalFrameShifts_S	totalInsertedNucs_L	totalInsertedNucs_M	totalInsertedNucs_S	totalSnps_L	totalSnps_M	totalSnps_S	totalStopCodons	totalUnknownNucs_L	totalUnknownNucs_M	totalUnknownNucs_S	travelHistory	annotations
```

Comment:
@fhennig what is the last metadata entry?
```
[{""fileId"":""7960e7ac-1282-4b39-8ca2-6c4a71e3fc30"",""name"":""LOC_000006N.embl"",""url"":""http://localhost:8079/files/get/LOC_000006N/1/annotations/LOC_000006N.embl""}]"
```
I didn't realize we needed this... I think it shouldn't be there - or am I missing sth about file upload?

Reply:
Ah! Yes, not needed, I removed it. That was only there because I downloaded the metadata file, and then it was in there.